packages/website/styles/faq.css

```diff
@@ -0,0 +1,16 @@
+.faq-item h2 {
+  font-size: 28px;
```

Comment:
Suggestion (non-blocking): Some of the titles on mobile are quite big still. Consider adding another breakpoint or using a responsive font-size function e.g. clamp to dynamically size it.
<img width="427" alt="Screen Shot 2022-03-02 at 9 34 04 AM" src="https://user-images.githubusercontent.com/1189523/156416628-e58f0852-cfb9-43b0-8ad9-117557c13493.png">